torch/csrc/cuda/Module.cpp

```diff
@@ -1203,6 +1203,25 @@ static void registerCudaPluggableAllocator(PyObject* module) {
     return c10::cuda::CUDACachingAllocator::getCheckpointState(device, id);
   });
 
+  m.def(
+      "_cuda_beginAllocateCurrentStreamToPool",
+      [](int device, at::cuda::MempoolId_t mempool_id) {
```

Comment:
nit: feels weird to use the default stream but explicit device vs (default device + default stream) or (explicit stream)

Reply:
True. Is there any way of passing a python object as `CudaStream_t` ?